content/posts/2018-01-10-kotlin-null.md

```diff
@@ -0,0 +1,249 @@
+---
+title: "Kotlin: The Problem with `null`"
+date: 2018-01-10
+slug: kotlin-the-problem-with-null
+---
+
+I think almost everybody in the CS world knows about the [Null References: The Billion Dollar Mistake](https://www.infoq.com/presentations/Null-References-The-Billion-Dollar-Mistake-Tony-Hoare) talk.
+However, it does not really matter if a concept of `null` references a good idea or a really bad one.
+We live in the world where we just have to deal with `null` this way or another.
+Yep, just deal with it :sunglasses:. So let’s ask the “How” question instead of the “Why” one.
```

Comment:
I'd remove emoji, at least on Android (yes, I'm reviewing from phone) it looks out of place

Reply:
Yeah, I’m going to remove and feel the same way. I should work on the language to avoid too much formalities in the future.